test/models/network.js

```diff
@@ -10,6 +10,7 @@ describe("Network", function() {
 	describe("#export()", function() {
 		it("should produce an valid object", function() {
 			var network = new Network({
+				uuid: "hello world",
```

Comment:
Don't we want actually valid UUIDs there, even in test objects?

Reply:
This is testing that it's being preserved. There's a test down below to test actual generation.

Reply:
Right, I just thought it'd be better to keep an actually valid value but whatever, no real argument there.